ggml/src/ggml-cuda/quantize.cu

```diff
@@ -47,6 +47,131 @@ static __global__ void quantize_q8_1(
     y[ib].ds = make_half2(d, sum);
 }
 
+__device__ __forceinline__ uint8_t compute_e8m0_scale(float amax) {
+    if (!(amax > 0.0f)) {
+        return 0;
+    }
+
+    // FP4 E2M1: max exponent (unbiased) is 2.
+    constexpr int FP4_E2M1_EMAX = 2;
+
+    const float e = __log2f(amax);
```

Comment:
```suggestion
    const float e = log2f(amax);
```

This should fix the MUSA CI and since we are compiling with `-fast_math` anyways it should not make a difference for CUDA.